teku/src/main/java/tech/pegasys/teku/cli/deposit/DepositGenerateAndRegisterCommand.java

```diff
@@ -50,11 +50,9 @@
   @Mixin private GenerateParams generateParams;
 
   @Option(
-      names = {"--Xconfirm-enabled"},
-      arity = "1",
-      defaultValue = "true",
+      names = {"--Xquiet"},
       hidden = true)
```

Comment:
nit: Not sure if this be a hidden option. Seems like a sensible option to have?

Reply:
Why did we rename it to not be compliant with our CLI style guide?  `--confirm-enabled` would have been what I expected it to be (and I agree it shouldn't be hidden).

Reply:
@ajsutton It was previously hidden hence I renamed it without following `-enabled` suffix. The flag was renamed to make the intent more clear as previously it was to only suppress the "confirmation" dialog, now it is also suppressing the additional verbose status log as well. 

If we want it to be a non-dev (i.e. non-hidden) option,  we would need a more generic option name with `-enabled` prefix. 

We may also change to have two options, one to enable/disable confirmation message, the other one to enable/disable verbose status message.

Reply:
I think keeping it simple and just the single option for this would be sufficient in my mind. All you want to do is make it optional provide a detailed output of exactly whats going on at each stage. What about `--verbose-output-enabled` with default to True?